docs/whatsnew/1.2.0/1_2_0.rst

```diff
@@ -34,6 +35,29 @@ You can now take into account the discretized nature of observed binding times o
 improve results by removing bias from the fitted lifetimes. Simply use `discrete_model=True` with
 :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.fit_binding_times()`.
 
+Plot `KymoTrack` Gaussian fit
+-----------------------------
+
+You can now easily plot the fitting results from :func:`~lumicks.pylake.refine_tracks_gaussian` for a particular scan line using
+:meth:`KymoTrack.plot_fit()<lumicks.pylake.kymotracker.kymotrack.KymoTrack.plot_fit>` or
+:meth:`KymoTrackGroup.plot_fit()<lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.plot_fit>`.
+
+.. figure:: group_plot_fit.png
+
+    Gaussian fits to signal peaks along a scan line of a tracked kymograph.
+
+Refine gaussian simultaneous
+----------------------------
+
+The new fitting mode `"simultaneous"` was introduced to :func:`~lumicks.pylake.refine_tracks_gaussian` which enforces optimization bounds between
+peak positions when two tracks are close together. This ensures that individual gaussians cannot switch positions between two
+tracks (which was a fault behavior that could occur using the previous "multiple" method). For more information see the :doc:`changelog</changelog>`.
```

Comment:
Is it `a fault behavior` or `faulty behavior`?

Reply:
oops, typo